lib/tasks/add_store_to_items.rake

```diff
@@ -0,0 +1,26 @@
+namespace :add_store_to_items do
+  desc "Creates initial store"
+  task create_initial_store: :environment do
+    if Store.find_by(name: "Little Shop of Funsies")
+      puts "Store already exists"
+    else
+      Store.create(name: "Little Shop of Funsies", status: "active")
+      puts "#{store.name} created"
+    end
+  end
+
+  desc "Adds store to items 1-1011 which are the items in the original store"
+  task associate_items: :environment do
+    items = Item.where(id: 1..1011)
```

Comment:
Just to note, Marshall and I verified that this items collection has 1011 elements (via #count method). This was enough evidence that all previously created items were included in this update.

Reply:
Item.where(id: 1..1011).count gives us the original 1011 items that were loaded from the rake import_dump:load.